openlibrary/plugins/openlibrary/js/edit.js

```diff
@@ -15,6 +15,18 @@ function update_len() {
     $('#excerpts-excerpt-len').html(2000 - len).css('color', color);
 }
 
+export function initEditRow(){
+    document.querySelector('#add_row_button').addEventListener('click', ()=>add_row('website'));
+}
+
+function add_row(name) {
+    const inputs = document.querySelector(`#clone_${name} #inputs`);
```

Comment:
Wouldn't `#clone_${name} input` and making use of insertAfter, insertBefore do the same thing here? Is the identifier necessary?